docs/releases/release-v23-0.md

```diff
@@ -56,25 +56,20 @@ Updates to simplify the unit names used within the node wallet and unit conversi
 
 ### Pending/Receivable term RPC updates
 
-There are various changes related to the switch from `pending` to `receivable` in RPC calls as noted above. **Although all changes are backwards compatible, switching to the term `receivable` is advised**.
+There are various changes related to the switch from `pending` to `receivable` in RPC calls as noted above. **Although all changes are backwards compatible, switching to the term `receivable` in these cases is advised**.
 
 There are two main types of changes: RPC call name changes and updates to keys in the call requests and responses.
 
 **RPC call name changes**
 
-* [`accounts_pending`](../commands/rpc-protocol.md#accounts_pending) replaced by [`accounts_receivable`](../commands/rpc-protocol.md#accounts_receivable)
 * [`pending`](../commands/rpc-protocol.md#pending) replaced by [`receivable`](../commands/rpc-protocol.md#receivable)
 * [`pending_exists`](../commands/rpc-protocol.md#pending_exists) replaced by [`receivable_exists`](../commands/rpc-protocol.md#receivable_exists)
-* [`search_pending`](../commands/rpc-protocol.md#search_pending) replaced by [`search_receivable`](../commands/rpc-protocol.md#search_receivable)
-* [`search_pending_all`](../commands/rpc-protocol.md#search_pending_all) replaced by [`search_receivable_all`](../commands/rpc-protocol.md#search_receivable_all)
-* [`wallet_pending`](../commands/rpc-protocol.md#wallet_pending) replaced by [`wallet_receivable`](../commands/rpc-protocol.md#wallet_receivable)
 
 **Response/request key changes only**
 
 * [`account_balance`](../commands/rpc-protocol.md#account_balance)
 * [`account_info`](../commands/rpc-protocol.md#account_info)
 * [`accounts_balances`](../commands/rpc-protocol.md#accounts_balances)
-* [`blocks_info`](../commands/rpc-protocol.md#blocks_info)
```

Comment:
This doesn't seem to be related to the pending/receivable change.

Reply:
See optional parameters on both account_info and blocks_info for how these are related. I realized that only part of the desired solution is implemented on this (response contains `receivable` when optional parameter `pending` is given) so it probably should be left in as a reference. RPC commands page reflects the proper behavior now and the related node repository ticket has been updated to ensure the right changes are made in the future.